### PR TITLE
FIX java.lang.UnsupportedClassVersionError:  org/openjdk/nashorn/api/scripting/NashornScriptEngineFactory has been compiled by a more recent version of the Java Runtime

### PR DESCRIPTION
**Pull request by vharseko**
after update https://github.com/OpenIdentityPlatform/commons/commit/eb81d9f17fddf1540585eef1dad0a0fff06a60dc#diff-51ae2e20cc5ee3d7455c42bad837a633863c7ae7d65282dc8715e8cbf27b4bfdR37

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -96,8 +96,6 @@
     <commons-logging-api.version>1.1</commons-logging-api.version>
     <!--         <esapiport.version>2013-12-04</esapiport.version> -->
     <geoip.version>2.0.0</geoip.version>
-    <groovy.version>2.5.8</groovy.version>
```
Comment (maximthomas) [inline pom.xml:99]: The good practice is to keep versions in properties

File: openam-scripting/src/main/java/org/forgerock/openam/scripting/StandardScriptEngineManager.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public final class StandardScriptEngineManager extends ScriptEngineManager {
      */
     @Inject
     public StandardScriptEngineManager() {
-
```
Comment (maximthomas) [inline openam-scripting/src/main/java/org/forgerock/openam/scripting/StandardScriptEngineManager.java:74]: added backward compartibility in the followin PR: https://github.com/OpenIdentityPlatform/commons/pull/62.
Allows to use [commons](https://github.com/OpenIdentityPlatform/commons) library in OpenAM without any modification.

Comment (vharseko) [reply openam-scripting/src/main/java/org/forgerock/openam/scripting/StandardScriptEngineManager.java:74]: have to support deploy for multiple JDKs ?

Comment (maximthomas) [reply openam-scripting/src/main/java/org/forgerock/openam/scripting/StandardScriptEngineManager.java:74]: Yes, this makes sense

**Pull request merged**